pkg/cli/build.go

```diff
@@ -35,7 +35,7 @@ type BuildOptions struct {
 
 const (
 	MaxPullPushRetries = 3
-	PullPushRetryDelay = 2 * time.Second
+	PullPushRetryDelay = "2s"
```

Comment:
Changing the type of this exported constant is an API break, and this one affects podman.

Reply:
@nalind ah i missed it, I will introduce a new constant for str version of delay. Just curious aren't we going to remove the usage of old constant from podman on next vendor of buildah ( but anyways not an excuse for breaking public API so i'll amend this ).

Reply:
Might be easier to use the value's `.String()` method as the default option value.